data/culture/haw.lua

```diff
@@ -0,0 +1,274 @@
+-- Source names: https://en.wiktionary.org/wiki/Appendix:Hawaiian_given_names
+-- Source general info: https://nameberry.com/blog/hawaiian-names
+-- Note: Hawaiian names are unisex even though some names are used predominantly for one of the sexes.
+-- Many of the given names are therefore represented on both lists. Also, some are present on all the lists
+-- since they are also used as surname.
+
+local CultureName = require './common'
+
+local male = {
+	'Ahe',
+	'Aʻiaʻi',
+	'Analu',
+	'Hali',
+	'Healani',
+	'Hoku',
+	'Iwalani',
+	'Kahanuola',
+	'Kai',
+	'Kaikane',
+	'Kaimi',
+	'Kaiea',
+	'Kaiko',
+	'Kaimana',
+	'Kainona',
+	'Kalani',
+	'Kalei',
+	'Kaleolani',
+	'Kalino',
+	'Kalua',
+	'Kanai',
+	'Kaniela',
+	'Kanoa',
+	'Kapueo',
+	'Kau',
+	'Kauai',
+	'Kaʻuhane',
+	'Kauʻionālani',
+	'Kawena',
+	'Kawai',
+	'Kawika',
+	'Keahilani',
+	'Kealakekua',
+	'Kealani',
+	'Kealoha',
+	'Keani',
+	'Keanu',
+	'Keikilani',
+	'Keimoni',
+	'Kekoa',
+	'Kelani',
+	'Keola',
+	'Keone',
+	'Keyon',
+	'Khalei',
+	'Kimo',
+	'Kimokea',
+	'Koa',
+	'Koalani',
+	'Konane',
+	'Kulani',
+	'Ky',
+	'Lihau',
+	'Lono',
+	'makai',
+	'Makani',
+	'Makoa',
+	'Malulani',
+	'Mana',
+	'Maui',
+	'Moani',
+	'Momilani',
+	'Nalu',
+	'Nahele',
+	'Nainoa',
+	'Pika',
+	'Pomaikai',
+	'Pohaku',
+	'Oke',
+	'Wailani',
+	'Wanaao',
+}
+
+local female = {
+	'Ahe',
+	'Aʻiaʻi',
+	'Hali',
+	'Hana',
+	'Haunani',
+	'Healani',
+	'Hoku',
+	'Iwalani',
+	'Kahanuola',
+	'Kai',
+	'Kaimi',
+	'Kaiea',
+	'Kaiko',
+	'Kaimana',
+	'Kainona',
+	'Kalani',
+	'kalehua',
+	'Kalei',
+	'Kaleolani',
+	'Kalua',
+	'Kamea',
+	'Kaniela',
+	'Kapueo',
+	'Kau',
+	'Kauai',
+	'Kaʻuhane',
+	'Kauʻionālani',
+	'Kawena',
+	'Kawai',
+	'Keahilani',
+	'Kealakekua',
+	'Kealani',
+	'Kealoha',
+	'Keani',
+	'Keikilani',
+	'Kelani',
+	'Keone',
+	'Khalei',
+	'Kiana',
+	'Kimokea',
+	'Koalani',
+	'Konane',
+	'Kulani',
+	'Kuʻuipo',
+	'Ky',
+	'Lana',
+	'Lei',
+	'Leinani',
+	'Lihau',
+	'Lilo', -- ;)
+	'Lono',
+	'luana',
+	'Makani',
+	'Malia',
+	'Malulani',
+	'Mana',
+	'Mele',
+	'Moana',
+	'Moani',
+	'Momilani',
+	'Nalu',
+	'Nahele',
+	'Nani',
+	'Pika',
+	'Pomaikai',
+	'Pohaku',
+	'Oke',
+	'Wailani',
+	'Wanaao',
+}
+
+local surname = {
+	'Ailana',
+	'Aka',
+	'Akamai',
+	'Akamu',
+	'Akana',
+	'Akina',
+	'Alama',
+	'Alana',
+	'Alika',
+	'Alohi',
+	'Anela',
+	'Aukai',
+	'Ele',
+	'Etana',
+	'Eusebio',
+	'Haku',
+	'Hawea',
+	'Hale',
+	'Halia',
+	'Haoa',
+	'Haukea',
+	'Havika',
+	'Hekekia',
+	'Hikialani',
+	'Ikaika',
+	'Illima',
+	'Inoa',
+	'Inoke',
+	'Iokua',
+	'Iona',
+	'Ionakana',
+	'Iosua',
+	'Isaia',
+	'Kaʻanaʻana',
+	'Kaʻaukai',
+	'Kahale',
+	'Kahananui',
+	'Kahele',
+	'Kahie',
+	'Kahue',
+	'Kai',
+	'Kainoa',
+	'Kaiwi',
+	'Kalama',
+	'Kalani',
+	'Kalawaiʻa',
+	'Kalili',
+	'Kalua',
+	'Kama',
+	'Kamealoha',
+	'Kane',
+	'Kapule',
+	'Kaʻuhane',
+	'Kawai',
+	'Kawena',
+	'Keahi',
+	'Keaka',
+	'Kealani',
+	'Kealoha',
+	'Keawe',
+	'Kekoa',
+	'Keliʻi',
+	'Keona',
+	'Konani',
+	'Lani',
+	'Lee',
+	'Leilani',
+	'Leimomi',
+	'Leo',
+	'Likeke',
+	'Lilo',
```

Comment:
Note that this could create a name e.g. 'Lilo Lilo'. Is this well formed?

Reply:
Yes. I believe this is how it's done.